doc/libpmemset/pmemset_source_pread_mcsafe.3.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: manual
+Content-Style: 'text/css'
+title: _MP(PMEMSET_SOURCE_PREAD_MCSAFE, 3)
+collection: libpmemset
+header: PMDK
+date: pmemset API version 1.0
+...
+
+[comment]: <> (SPDX-License-Identifier: BSD-3-Clause)
+[comment]: <> (Copyright 2021, Intel Corporation)
+
+[comment]: <> (pmemset_source_pread_mcsafe.3 -- man page for libpmemset machine safe read/write operations)
+
+[NAME](#name)<br />
+[SYNOPSIS](#synopsis)<br />
+[DESCRIPTION](#description)<br />
+[RETURN VALUE](#return-value)<br />
+[ERRORS](#errors)<br />
+[SEE ALSO](#see-also)<br />
+
+# NAME #
+
+**pmemset_source_pread_mcsafe**(), **pmemset_source_pwrite_mcsafe**() - read source
+contents or write to the source in a safe manner
+
+# SYNOPSIS #
+
+```c
+#include <libpmemset.h>
+
+struct pmemset_source;
+int pmemset_source_pread_mcsafe(struct pmemset_source *src, void *buf,
+		size_t size, size_t offset);
+int pmemset_source_pwrite_mcsafe(struct pmemset_source *src, void *buf,
+		size_t size, size_t offset);
+```
+
+# DESCRIPTION #
+
+The **pmemset_source_pread_mcsafe**() function reads *size* bytes from the source *src*
+starting at offset *offset* into the buffer *buf*.
+The **pmemset_source_pwrite_mcsafe**() function writes *size* bytes from the buffer *buf*
+to the source *src* starting at the offset *offset*.
+
+Safe manner refers to the capability to handle errors resulting from bad blocks and
+returning them to the user. For details, see corresponding **libpmem2**(7) functions
```

Comment:
this reads as if this function somehow returns bad blocks. And "handle" is ambiguous.